scipy/stats/_continuous_distns.py

```diff
@@ -2818,22 +2819,16 @@ def _fitstart(self, data):
 
     # Trivial override of the fit method, so we can monkey-patch its
     # docstring.
+    @extend_notes_in_docstring(rv_continuous, notes="""\
```

Comment:
Might as well use the same decorator that's used elsewhere.  Doesn't really matter, since this doesn't even render.